code/game/machinery/slotmachine.dm

```diff
@@ -24,31 +24,41 @@
 	density = TRUE
 	circuit = /obj/item/circuitboard/computer/slot_machine
 	light_color = LIGHT_COLOR_BROWN
-	interaction_flags_machine = INTERACT_MACHINE_ALLOW_SILICON|INTERACT_MACHINE_SET_MACHINE // don't need to be literate to play slots
+	interaction_flags_machine = INTERACT_MACHINE_ALLOW_SILICON
 	var/money = 3000 //How much money it has CONSUMED
 	var/plays = 0
 	var/working = FALSE
 	var/balance = 0 //How much money is in the machine, ready to be CONSUMED.
 	var/jackpots = 0
 	var/paymode = HOLOCHIP //toggles between HOLOCHIP/COIN, defined above
 	var/cointype = /obj/item/coin/iron //default cointype
+	/// Icons that can be displayed by the slot machine.
+	var/static/list/icons = list(
+		FA_ICON_LEMON = list("value" = 2, "colour" = "yellow"),
+		FA_ICON_STAR = list("value" = 2, "colour" = "yellow"),
+		FA_ICON_BOMB = list("value" = 2, "colour" = "red"),
+		FA_ICON_BIOHAZARD = list("value" = 2, "colour" = "green"),
+		FA_ICON_APPLE_WHOLE = list("value" = 2, "colour" = "red"),
+		SEVEN = list("value" = 1, "colour" = "yellow"),
```

Comment:
``FA_ICON_7``

Reply:
Honestly you can maybe use local defines like this if you want to be really nitty

``#define JACKPOT_SEVENTS FA_ICON_7`` so you can use a define that makes more sense while still using the proper defines. Same for bomb since you gave that its own interaction.
Up to you.